src/WebJobs.Extensions.ApiHub/Helper/ApiHubHelper.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Azure.ApiHub.Management;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ApiHub
+{
+    public static class ApiHubHelper
```

Comment:
This probably shouldn't be public? You need to add a test to https://github.com/Azure/azure-webjobs-sdk-extensions/blob/master/test/WebJobs.Extensions.Tests/PublicSurfaceTests.cs following the example of the others, and review your surface area.